.github/libs/GithubUtils.js

```diff
@@ -143,8 +144,7 @@ class GithubUtils {
                 isAccessible: match[4] === 'x',
             }),
         );
-        const internalQAPRList = this.getStagingDeployCashInternalQA(issue);
-        return _.sortBy(_.union(PRList, internalQAPRList), 'number');
+        return _.sortBy(PRList, 'number');
```

Comment:
Actually, this is what was causing the internalQA PRs to be added to the normal PR list as well as the internal QA section. It is not necessary to be added here since the PRs are already fetched and then filtered somewhere else. This should fix that bug.